src/body.js

```diff
@@ -295,13 +297,23 @@ export const extractContentType = (body, request) => {
 		return null;
 	}
 
+	if (isFormData(body)) {
+		return `multipart/form-data; boundary=${request[INTERNALS].boundary}`;
+	}
+
 	// Detect form data input from form-data module
 	if (body && typeof body.getBoundary === 'function') {
-		return `multipart/form-data;boundary=${body.getBoundary()}`;
-	}
+		if (!warnedUsingOldFormData) {
+			console.warn('\u001B[33m%s\u001B[0m', '[WARN]', `
```

Comment:
I think, you can use [`util.deprecate`](https://nodejs.org/dist/latest-v16.x/docs/api/util.html#util_util_deprecate_fn_msg_code) here if this is deprecation warning

Reply:
When i googled it it seem like they wanted to deprecate util.deprecate

Reply:
It looks like they didn't. See: https://github.com/nodejs/node/pull/12631

Reply:
I saw that